packages/devtools_app/lib/src/flutter/utils.dart

```diff
@@ -14,3 +15,26 @@ Future<void> launchUrl(String url, BuildContext context) async {
     Notifications.of(context).push('Unable to open $url.');
   }
 }
+
+List<TextSpan> maybeConvertToAnsiText(String input, TextStyle defaultStyle) {
+  return decodeAnsiColorEscapeCodes(input, AnsiUp())
+      .map((entry) => TextSpan(
+            text: entry.text,
+            style: entry.style.isEmpty
+                ? defaultStyle
+                : TextStyle(
+                    color: entry.fgColor != null && entry.fgColor.length > 2
+                        ? Color.fromRGBO(entry.fgColor[0], entry.fgColor[1],
```

Comment:
add helper method to convert an array[int] to a Color object and use it for both cases.
Make it an assert the entry.fooColor.length == 3. It is a bug in AnsiUp if that isn't the case.